### ISSUE TITLE
Add running mean/var support to LayerNorm

### ISSUE THREAD

**Issue by ljjsalt** [pytorch/pytorch#84225]
### 🚀 The feature, motivation and pitch

Detach the LayerNorm from input during inference, so it works like BatchNorm that mean and var are not calculated during inference.

LayerNorm is time consuming and is not applicable in some applications. And from experiments that using running mean/var to perform inference only causes slightly downgrading performance.

I can work on this feature.

**Comment by ljjsalt** [comment]
Closed, because performance is not good as expected.
